lib/pages/login/login_page.dart

```diff
@@ -0,0 +1,245 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:intl_phone_field/intl_phone_field.dart';
+
+class LoginPage extends StatefulWidget {
+  static const String routeName = '/loginpage';
+
+  const LoginPage({super.key});
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  //form key
+  final _formKey = GlobalKey<FormState>();
+
+  //image variables
+  File? _image;
+  ImagePicker picker = ImagePicker();
+
+  //function to click photo from camera
+  _imgFromCamera() async {
+    XFile? image =
+        await picker.pickImage(source: ImageSource.camera, imageQuality: 50);
+
+    setState(() {
+      _image = File(image!.path);
+    });
+  }
+
+  //function to choose image from gallery
+  _imgFromGallery() async {
+    XFile? image =
+        await picker.pickImage(source: ImageSource.gallery, imageQuality: 50);
+
+    setState(() {
+      _image = File(image!.path);
+    });
+  }
+
+  //function to show image picker
+  void _showPicker(context) {
+    showModalBottomSheet(
+        context: context,
+        builder: (BuildContext bc) {
+          return SafeArea(
+            child: Wrap(
+              children: <Widget>[
+                ListTile(
+                    leading: const Icon(Icons.photo_library),
+                    title: const Text('Photo Library'),
+                    onTap: () {
+                      _imgFromGallery();
```

Comment:
maybe await till the image is updated
totally fine the current system

just test it once with await